.travis.yml

```diff
@@ -11,6 +11,13 @@ before_install:
   # Needed to deploy pull request and releases
   - sudo apt-get -y install python-pip python-dev
   - pip install awscli --upgrade --user
+  # use production url for price estimator on master or tagged
+  - export PRICE_ESTIMATOR_URL=$([[
```

Comment:
That's fancy! I was thinking something way simpler, but hey, glad you are doing this :)